docs/connect/ide.md

```diff
@@ -6,6 +6,29 @@ Mostly through its PostgreSQL interface, CrateDB supports working with popular
 database IDE (Integrated Development Environment) applications.
 
 
+## DataGrip
+
+```{div}
+:style: "float: right"
+[![](https://blog.jetbrains.com/wp-content/uploads/2019/01/datagrip_icon.svg){w=120px}](https://www.jetbrains.com/datagrip/)
+```
+
+[DataGrip] is a cross-platform database IDE that is tailored to suit the specific needs
+of professional SQL developers.
+
+Connecting DataGrip to CrateDB uses the vanilla PostgreSQL JDBC Driver,
+the blog article [Blog: Use CrateDB With DataGrip] explains how it works.
+
+![image](https://19927462.fs1.hubspotusercontent-na1.net/hub/19927462/hubfs/13-Datagrip.png?width=1536&name=CrateDB-DataGrip.png){h=200px}
+![image](https://www.pgadmin.org/static/COMPILED/assets/img/screenshots/pgadmin4-welcome-light.png){h=200px}
+
+:::{caution}
+Please note while the query console works well, you will notice that many UI
+actions do not work yet. We are tracking corresponding gaps at [Tool: DataGrip],
+and appreciate any contributions to improve the situation.
+:::
```

Comment:
@surister: I've just added a corresponding note, as you suggested. Thanks again!